### DISCUSSION TITLE
skoala/standard/monitor/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1919] [Announcements]
# skoala/standard/monitor/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/standard/monitor/

**Comment by wilsonwu** [comment]
缺少sentinel接入的配置更新：https://github.com/openinsight-proj/opentelemetry-demo/blob/daocloud/src/dataservice/src/main/resources/bootstrap.yml，参考此demo。
